### PR TITLE
appboy-web-sdk v3 support

### PR DESCRIPTION
**Pull request by davidbielik**
**What does this PR do?**

Adds support for version `3.1` in the `appboy` (braze) Web SDK integration.

Existing customers on older options (`1`, `2`, `2.7`) will continue to load those respective SDK versions. Customers who choose the new `3.1` option will load the `3.1` version of Braze SDK.

**Are there breaking changes in this PR?**

No

**Testing**

Added a new test for `initializeV3`

**`yarn test`**
image

**Any background context you want to provide?**

**Is there parity with the server-side/android/iOS integration components (if applicable)?**
n/a

**Does this require a new integration setting? If so, please explain how the new setting works**
Yes, a new version in the dropdown for `3.1`

**Links to helpful docs and other external resources**

https://github.com/Appboy/appboy-web-sdk/blob/master/CHANGELOG.md#300

### CODE REVIEW

**Review by wesleyorbin — APPROVED** [review]

File: integrations/appboy/HISTORY.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+1.16.0 / 2020-12-07
+
```
Comment (mericsson) [inline integrations/appboy/HISTORY.md:2]:
```suggestion
1.16.0 / 2020-12-07
===================
```

**Comment by mericsson** [comment]
@davidbielik the change lgtm - Can you fix conflicts that have arisen since you opened PR (sorry about that). Once fixed, I'll merge and deploy to your test account so you can verify expected behavior. Can you share via email your segment source I should enable? Thanks!

File: integrations/appboy/HISTORY.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+1.16.0 / 2020-12-07
```
Comment (davidbielik) [inline integrations/appboy/HISTORY.md:1]:
```suggestion
1.16.0 / 2020-12-16
```

**Comment by mericsson** [comment]
Merged here https://github.com/segmentio/analytics.js-integrations/pull/542

**Pull request closed without merging**